app/facades/movie_facade.rb

```diff
@@ -5,6 +5,11 @@ def self.top_40
     convert_to_objects(movies)
   end
 
+  def self.upcoming_movies
+    movies = MovieService.now_playing
+    convert_to_objects(movies[:results].sample(4))
+  end
+
```

Comment:
:results returns 20 movie objects. The sample will display a random set when user refreshes the page.